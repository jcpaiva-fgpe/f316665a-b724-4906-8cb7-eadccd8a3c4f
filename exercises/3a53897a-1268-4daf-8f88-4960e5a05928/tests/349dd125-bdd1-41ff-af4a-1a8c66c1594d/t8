-40000 -25000 4000 25000
10
1000 1000
-24 0
0 1000
0 300
12 12
2 -2
2 0
0 2
-20 -20
-20 20
