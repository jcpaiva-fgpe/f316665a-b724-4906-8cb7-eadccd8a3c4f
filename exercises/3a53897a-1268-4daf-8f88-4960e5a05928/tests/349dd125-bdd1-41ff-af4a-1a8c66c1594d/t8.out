7
Barcelona 10
Frankfurt 10
Funchal 10
Lisboa 10
Madrid 10
Paris 8
Porto 10
